pluginClientSdkLib/src/main/java/com/twofortyfouram/locale/sdk/client/receiver/AbstractAsyncReceiver.java

```diff
@@ -62,6 +62,7 @@
                 new Pair<PendingResult, AsyncCallback>(pendingResult, callback)));
     }
 
+    @SuppressWarnings("unchecked")
```

Comment:
That works too.  I don't think we've hit over-engineering until there's at least unit test for that private helper method :-)